Kiosk/ServiceHelpers/InkRecognizer.cs

```diff
@@ -0,0 +1,139 @@
+// distribute, sublicense, and/or sell copies of the Software, and to
+// permit persons to whom the Software is furnished to do so, subject to
+// the following conditions:
+// 
+// The above copyright notice and this permission notice shall be
+// included in all copies or substantial portions of the Software.
+// 
+// THE SOFTWARE IS PROVIDED ""AS IS"", WITHOUT WARRANTY OF ANY KIND,
+// EXPRESS OR IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF
+// MERCHANTABILITY, FITNESS FOR A PARTICULAR PURPOSE AND
+// NONINFRINGEMENT. IN NO EVENT SHALL THE AUTHORS OR COPYRIGHT HOLDERS BE
+// LIABLE FOR ANY CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION
+// OF CONTRACT, TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION
+// WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
+// 
+
+using System;
+using System.Collections.Generic;
+using System.Net.Http;
+using System.Numerics;
+using System.Text;
+using System.Threading.Tasks;
+using Windows.Data.Json;
+using Windows.Graphics.Display;
+using Windows.UI.Input.Inking;
+
+namespace ServiceHelpers
+{
+    public class InkRecognizer
+    {
+        string inkRecognitionUrl;
+        HttpClient httpClient;
+
+        private IDictionary<uint, InkStroke> StrokeMap { get; set; }
+        private string LanguageCode { get; set; }
+
+        public InkRecognizer(string subscriptionKey, string endpoint, string inkRecognitionUrl)
+        {
+            httpClient = new HttpClient() { BaseAddress = new Uri(endpoint) };
+            httpClient.DefaultRequestHeaders.Add("Ocp-Apim-Subscription-Key", subscriptionKey);
+
+            this.inkRecognitionUrl = inkRecognitionUrl;
+
+            StrokeMap = new Dictionary<uint, InkStroke>();
+            LanguageCode = "en-US";
+        }
+
+        public void AddStrokes(IReadOnlyList<InkStroke> strokes)
+        {
+            foreach (var stroke in strokes)
+            {
+                StrokeMap[stroke.Id] = stroke;
+            }
+        }
+
+        public void ClearStrokes()
+        {
+            StrokeMap.Clear();
+        }
+
+        public void SetLanguage(string languageCode)
+        {
+            LanguageCode = languageCode;
+        }
+
+        public JsonObject ConvertInkToJson()
+        {
+            // If needed to use the device's DPI, and example is below. Whatever DPI is used below will need to be used again when handling the response's ink points.
+            //var displayInformation = DisplayInformation.GetForCurrentView();
+            //float dpi = displayInformation.LogicalDpi;
+            //float dipsPerMm = dpi / 25.4f;
+
+            float dipsPerMm = 96 / 25.4f;
+
+            var payload = new JsonObject();
+            var strokesArray = new JsonArray();
+
+            foreach (var stroke in StrokeMap.Values)
+            {
+                var jStroke = new JsonObject();
+                var pointsCollection = stroke.GetInkPoints();
```

Comment:
>var [](start = 16, length = 3)

your usage of var here and below is a little confusing.  It should be 100% clear what type you are getting back.  I don't think it is here nor below.   that or it should greatly increase readability if the variable type is crazy complex.